### PR TITLE
Add filter by store id to outbound count

### PR DESCRIPTION
**Pull request by roxy-dao**
Outbound count was showing all outbound invoices instead of filtering by store

### CODE REVIEW

**Review by mark-prins — APPROVED** [review]
👍  ah! thanks!!

File: server/service/src/dashboard/invoice_count.rs [hunk 1]
```diff
@@ -157,6 +157,7 @@ impl InvoiceCountServiceTrait for InvoiceCountService {
         let repo = InvoiceRepository::new(&ctx.connection);
         Ok(repo.count(Some(
             InvoiceFilter::new()
+                .store_id(EqualFilter::equal_to(&ctx.store_id))
                 .r#type(InvoiceRowType::OutboundShipment.equal_to())
                 .status(InvoiceRowStatus::Picked.equal_to()),
```
Comment (mark-prins) [inline server/service/src/dashboard/invoice_count.rs:162]: am wondering about our logic here.. the heading on the dashboard is 'Shipments to be picked' but the status is already 'Picked' 🤔

Comment (roxy-dao) [reply server/service/src/dashboard/invoice_count.rs:162]: Just saw that too after when I went back to dashboard 👀

Comment (mark-prins) [reply server/service/src/dashboard/invoice_count.rs]: thanks! I see that there's an issue with caching now. Just putting in a fix for that - it's not essential, it just means that when you switch stores the numbers don't update as they should

Comment (mark-prins) [reply server/service/src/dashboard/invoice_count.rs]: have just snuck in this one 8b5166966 😉

**Pull request merged**